src/utils/utils.js

```diff
@@ -2,3 +2,12 @@ export const formatDate = dateString => {
   const date = new Date(dateString)
   return `${date.getDate()}.${date.getMonth() + 1}.${date.getFullYear()}`
 }
+
+export const getLangCode = slug => {
+  const regex = /(\bfi)|(\ben)|(\bsv)/g
+  const regexedLangCode = slug.match(regex)
+  // Default to en if lang code is not found
+  const langCode = regexedLangCode ? regexedLangCode[0] : "en"
```

Comment:
Yes, I see your point, that could be a problem. However, we actually can expect it to be the first one, as the urls are always `/langCode/blablabla`. Also, we need to return the lang code (without slashes), they should be stripped away if the match was with slashes. So maybe just stick to this as we can expect the first match to be the correct one 😊

Reply:
And of course I reread the comment and see that I missed the most important part 😅

Reply:
Now that I tried to do some searching, I couldn't confirm either that the array keeps or does not keep the order. I'll refactor this to be without the global flag, so that it returns the first match only